shared/constants/chat2/message.js

```diff
@@ -1067,6 +1067,28 @@ export const enoughTimeBetweenMessages = (
       message.timestamp - previous.timestamp > howLongBetweenTimestampsMs
   )
 
+export const shouldShowPopup = (message: Types.Message) => {
+  switch (message.type) {
+    case 'text':
+    case 'attachment':
+    case 'requestPayment':
+      return true
+    case 'sendPayment': {
+      // Is the payment pending?
+      if (
+        message &&
+        message.paymentInfo &&
```

Comment:
The `paymentInfo` won't always be on the message. This should use `Constants.getPaymentMessageInfo` to get the info.